user-management/src/main/elm/sources/UserManagement.elm

```diff
@@ -82,11 +79,16 @@ update msg model =
         ToastyMsg subMsg ->
             Toasty.update defaultConfig ToastyMsg subMsg model
 
-        ActivePanelSettings username->
-            ({model | editMod = On, userFocusOn = username}, Cmd.none)
+        ActivePanelAddUser ->
+            if model.addMod == On then
+                ({model | addMod = Off}, Cmd.none)
+            else
+                ({model | addMod = On, editMod = Off}, Cmd.none)
+        ActivePanelSettings username ->
+            ({model | addMod = Off, editMod = On, userFocusOn = username}, Cmd.none)
```

Comment:
maybe we could have the same thing for to close modification if you click on the same user name

```
if editMod = On and userFocusOn = username
  editMode = Off
else 
  on
```

Reply:
I think you also need to change you model so you don't have addMod and EditMod but a panelMode of type PanelMode that has 3 value : Closed | AddMode | EditMode String (the username we modify)

Reply:
This will allow to remove from model the "userFocusOn"